src/main/java/org/tikv/raw/RawKVClient.java

```diff
@@ -129,8 +131,17 @@ public void putAtomic(ByteString key, ByteString value, long ttl) {
   private void put(ByteString key, ByteString value, long ttl, boolean atomic) {
     String label = "client_raw_put";
     Histogram.Timer requestTimer = RAW_REQUEST_LATENCY.labels(label).startTimer();
+    SlowLog slowLog =
+        new SlowLogImpl(
+            new HashMap<String, String>(8) {
+              {
+                put("func", "put");
+                put("key", Arrays.toString(key.toByteArray()));
```

Comment:
we should redact 'key' when log redaction is enabled.

Reply:
why should we log the key, if redaction is enabled？